src/formatters/IndentFormatter.ts

```diff
@@ -90,6 +91,17 @@ export class IndentFormatter {
                     continue;
                 }
 
+                //skip indent for 'function'|'sub' used as type if another indent already exist in this line
+                if (
+                    CallableKeywordTokenKinds.includes(token.kind) &&
```

Comment:
I think this should be a little more generic. There's a whole set of indentors and outdentors and intermediary items. Let's use those lists instead of just fixing this for `function` and `sub`.

Reply:
updated